src/Http/Http.Extensions/gen/RequestDelegateGeneratorSources.cs

```diff
@@ -42,6 +44,32 @@ public GeneratedProducesResponseTypeMetadata(Type? type, int statusCode, string[
 
         public IEnumerable<string> ContentTypes { get; }
     }
+
+    file sealed class GeneratedAcceptsMetadata : IAcceptsMetadata
```

Comment:
Nit: Put this in a separate variable and generate it conditional on the existence of a request body.

Methods like the one below would need response metadata but not request body metadata.

```csharp
app.MapGet("/todos/{id}", (int id) => todos.GetTodo(id)
```

Reply:
Done!